src/libs/actions/IOU.js

```diff
@@ -1070,6 +1070,11 @@ function getSendMoneyParams(report, amount, currency, comment, paymentMethodType
         optimisticData.push(optimisticPersonalDetailListData);
     }
 
+    let reportPreviewAction = ReportActionsUtils.getReportPreviewAction(chatReport.reportID, optimisticIOUReport.reportID);
+    if (!reportPreviewAction) {
+        reportPreviewAction = ReportUtils.buildOptimisticReportPreview(chatReport.reportID, optimisticIOUReport.reportID);
+    }
```

Comment:
This seems unrelated to the profile changes 🤔

Reply:
Oopsie! 😄